sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/ProcessBundleHandler.java

```diff
@@ -931,6 +967,17 @@ synchronized String getInstructionId() {
       return this.instructionId;
     }
 
+    synchronized List<CacheToken> getCacheTokens() {
+      return this.cacheTokens;
+    }
+
+    synchronized Cache<Object, Object> getBundleCache() {
+      if (this.bundleCache == null) {
```

Comment:
this bundleCache doesn't seem to get reset to null after bundle finishes, it will use the same instructionId space right?

Reply:
PTAL